http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/LeoAppServiceInterp.scala

```diff
@@ -1305,6 +1305,13 @@ final class LeoAppServiceInterp[F[_]: Parallel](config: AppServiceConfig,
         if (req.appType == AppType.Allowed)
           Some(config.leoKubernetesConfig.allowedAppConfig.numOfReplicas)
         else None
+      autodeleteEnabled = req.autodeleteEnabled.getOrElse(false)
+      autodeleteThresholdMin = req.autodeleteThresholdMin.getOrElse(0)
```

Comment:
```suggestion
      autodeleteThreshold = req.autodeleteThreshold.getOrElse(0)
```

Reply:
... I am wondering if it makes more sense to not set this variable because it could present some confusion as to why we are setting a variable and then not using it. my worry is that we pass back to our persistence layer the default of (0) on line 1342 versus treating it as null/None if not set. Thoughts?